test/evergreen.yml

```diff
@@ -3620,6 +3620,22 @@ tasks:
             ${PREPARE_TEST_ENV}
             ./test_model.sh
 
+  - name: model-test-long
+    tags: ["model_checking"]
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
```

Comment:
There is already `-DENABLE_MODEL`, see `test/CMakeLists.txt`. It is just not yet used in `evergreen.yml`, because there wasn't a need to disable it yet.

Reply:
Cool, thanks for pointing that out.